flow360/component/utils.py

```diff
@@ -68,7 +68,7 @@ def is_valid_uuid(id, allow_none=False, valid_prefixes=None):
     Checks if id is valid
     """
     if valid_prefixes is None:
-        valid_prefixes = ["folder-", "g-", "geo-", "sm-", "vm-", "c-"]
+        valid_prefixes = ["folder-", "g-", "geo-", "sm-", "vm-", "c-", "prj-", "dft-"]
```

Comment:
along this PR, can use also do this https://flow360.atlassian.net/browse/SCFD-2262
so we can remove prefixes? Backend prefers to have more freedom on these

Reply:
Done.